pontoon/base/models.py

```diff
@@ -1087,7 +1087,7 @@ def with_status_counts(self, locale):
 
     def missing(self, locale):
         return self.with_status_counts(locale).filter(
-            Q(approved_count=0) & Q(fuzzy_count=0) & Q(suggested_count=0)
+            Q(approved_count__lt=F('expected_count')) & Q(fuzzy_count__lt=F('expected_count')) & Q(suggested_count__lt=F('expected_count'))
```

Comment:
Are you sure?
https://github.com/mozilla/pontoon/blob/master/pontoon/base/tests/test_models.py#L1079